anaSBitMonitor.py

```diff
@@ -40,6 +42,16 @@
     isValidValues = np.unique(initInfo['isValid'])
     ratesUsed = np.unique(initInfo['ratePulsed'])
 
+    ##### NEED TO FIX
+    #### VERY TEMPORARY
+    from gempython.gemplotting.mapping.chamberInfo import gemTypeMapping
+    if 'gemType' not in inF.sbitDataTree.GetListOfBranches():
+        gemType = "ge21"
```

Comment:
I put these comments because I was unsure the formatting of the gemType variable in the tree. I formatted it as a branch with the name "gemType" containing a number that mapped ge11 => 0, ge21 => 1. I didn't think this was how the final code would be done.

Is there a gemType branch implemented in the current tree structure, and if so, what is the format? I can change all these "TEMPORARY"s to that. 

Also, ge21 must have slipped by after some tests, will change

Reply:
OK, keep a `FIXME` text in each side of the comment block, so that it can easily be found when doing the eventual cleanup

Reply:
Done